README.md

```diff
@@ -0,0 +1,270 @@
+   free to reach out to the RFC assignee in particular to get help identifying
+   stakeholders and obstacles.
+1. All conversations should stay within the comment thread of the pull request
+   itself. Offline discussion will be summarized on the pull request comment
+   thread.
+1. RFCs rarely go through this process unchanged, especially as alternatives 
+   and drawbacks are shown. You can make edits, big and small, to the RFC to 
+   clarify or change the design, but make changes as new commits to the pull 
+   request, and leave a comment on the pull request explaining your changes. 
+   Specifically, do not squash or rebase commits after they are visible on the 
+   pull request.
+1. At some point, a maintainer will propose a "motion for final comment period" 
+   (FCP), along with a disposition for the RFC (merge, close, or postpone).
+    - This step is taken when enough of the tradeoffs have been discussed that 
+      the subteam is in a position to make a decision. That does not require 
+	  consensus amongst all participants in the RFC thread (which is usually 
+	  impossible). However, the argument supporting the disposition on the RFC 
+	  needs to have already been clearly articulated, and there should not be 
+      a strong consensus against that position outside of the maintainers.
+      Maintainers use their best judgment in taking this step, and the FCP 
+      itself ensures there is ample time and notification for stakeholders to 
+      push back if it is made prematurely.
+    - For RFCs with lengthy discussion, the motion to FCP is usually preceded by 
+      a summary comment trying to lay out the current state of the discussion 
+	  and major tradeoffs/points of disagreement.
+    - Before actually entering FCP, all members of the maintainers group must 
+      sign off; this is often the point at which many members first review the 
+	  RFC in full depth.
+1. Once an FCP is called, it will last ten calendar days, so it is open for at
+   least five business days. We will do our best to advertise widely in our
+   community channels. This way all stakeholders have a chance to lodge any 
+   final objections before a decision is reached.
+1. In most cases, the FCP period is quiet, and the RFC is either merged or
+   closed. However, sometimes substantial new arguments or ideas are raised. 
+   When this happens, the RFC is canceled and the it goes back into development 
+   mode.
+
+Life Cycle
+----------
+
+Once an RFC reaches "active" status, an issue will be created in the
+`scipian/community` repository and the label `type/epic` will be applied. Using 
+the RFC as reference, the epic can then be broken down into it's story's. Since
+we have multiple repository's in the Scipian organization, the story's should be
+created in the corresponding repository the story belongs to. When the work has 
+been broken down then the authors may implement it and follow the pull request 
```

Comment:
There should be a comma after `down`, e.g. `When the work has been broken down, then ...`